trunk/research/players/js/srs.page.js

```diff
@@ -116,6 +118,38 @@ function build_default_rtc_url(query) {
     return uri;
 };
 
+function build_default_whip_whep_url(query, apiPath) {
+    // The format for query string to overwrite configs of server.
+    console.log('?eip=x.x.x.x to overwrite candidate. 覆盖服务器candidate(外网IP)配置');
+    console.log('?api=x to overwrite WebRTC API(1985).');
+    console.log('?schema=http|https to overwrite WebRTC API protocol.');
+
+    var server = (!query.server)? window.location.hostname:query.server;
```

Comment:
why not give `?a:b` some spaces, ref line #131 `'http:' ? '1985' : '1990'`. They are so crowded, are they?